tests/test_genetic_value.py

```diff
@@ -98,20 +99,25 @@ def test_binary_mutation_model(self, num_ind):
         genetic_df = tstrait.calculate_genetic_value(ts, trait_df)
 
         assert len(genetic_df) == num_ind
+        # 3 columns, individual_id, genetic_value and trait_id
         assert genetic_df.shape[1] == 3
         assert np.array_equal(genetic_df["individual_id"], np.arange(num_ind))
         assert len(genetic_df["genetic_value"]) == num_ind
-        assert np.array_equal(genetic_df["trait_id"], np.zeros(num_ind))
+        assert np.all(genetic_df["trait_id"] == 0)
 
 
 class Test_genetic_value_input:
+    @pytest.fixture
```

Comment:
add @pytest.fixture(scope="class") to avoid invoking this more than once, when shared by multiple tests.